src/main/java/bisq/desktop/main/dao/proposal/ProposalListItem.java

```diff
@@ -62,13 +73,23 @@
     private TxConfidenceListener txConfidenceListener;
     private Tooltip tooltip = new Tooltip(Res.get("confidence.unknown"));
     private Transaction walletTransaction;
+    private ChangeListener<DaoPeriodService.Phase> phaseChangeListener;
+    private AutoTooltipButton actionButton;
+    private ImageView actionButtonIconView;
+    @Setter
+    private Runnable onRemoveHandler;
+    private Node actionNode;
 
-    public ProposalListItem(Proposal proposal,
-                            BsqWalletService bsqWalletService,
-                            BsqBlockChain bsqBlockChain,
-                            BsqBlockChainChangeDispatcher bsqBlockChainChangeDispatcher,
-                            BsqFormatter bsqFormatter) {
+    ProposalListItem(Proposal proposal,
+                     ProposalCollectionsManager proposalCollectionsManager,
+                     DaoPeriodService daoPeriodService,
+                     BsqWalletService bsqWalletService,
+                     BsqBlockChain bsqBlockChain,
```

Comment:
Is there a reason to use BsqBlockChain instead of the reader/writer classes here?

Reply:
Oh, not thats left over from refactoring. Should be ReadModel. Thanks, I will change it for next PR.